README.md

```diff
@@ -0,0 +1,186 @@
+# Todo
```

Comment:
One difference between replidraw-do and todo is that todo uses a single room. Dunno if we want to mention creating rooms here in the docs.